recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+
+# libcuda.so is a machine specific library. For building, we use
+# libcuda.so from stubs. For runtime, users must specify the location
+# of libcuda.so.1 in the environment variable LD_LIBRARY_PATH.
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib/stubs -Wl,-rpath-link,$PREFIX/lib/stubs"
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_LIBRARY_PATH=$PREFIX/lib/stubs"
+
+# Enforce PREFIX instead of BUILD_PREFIX:
+export ZLIB_ROOT=$PREFIX
+export LibArchive_ROOT=$PREFIX
+export Curses_ROOT=$PREFIX
+export Glog_ROOT=$PREFIX
+export Snappy_ROOT=$PREFIX
+export Boost_ROOT=$PREFIX
+export PNG_ROOT=$PREFIX
+export GDAL_ROOT=$PREFIX
+export BLOSC_ROOT=$PREFIX
+
+# Make sure -fPIC is not in CXXFLAGS (that some conda packages may
+# add):
+export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
+
+# Fixes https://github.com/Quansight/pearu-sandbox/issues/7
+#       https://github.com/omnisci/omniscidb/issues/374
+export CXXFLAGS="$CXXFLAGS -Dsecure_getenv=getenv"
+
+# Adjust OPENSSL_ROOT for conda environment. This ensures that
+# openssl is picked up from host environment:
+sed -i 's!/usr/local/opt/openssl!\'$PREFIX'!g' CMakeLists.txt
+
+# Avoid picking up boost/regexp header files from local system if
+# there:
+sed -i 's!/usr/local!\'$PREFIX'!g' CMakeLists.txt
+
+# Make sure that llvm-config and clang++ are from host environment,
+# otherwise UdfTest will fail:
+export PATH=$PREFIX/bin:$PATH
+
+# Fixes nvcc linker failure (nvcc assumes libraries are in
+# $PREFIX/lib64). Reconsider when new cudatoolk-dev is released:
+export LIBRARIES="$LIBRARIES -L$PREFIX/lib"
+
+# All these must be picked up from $PREFIX/bin
+CMAKE_ENV="CC=clang CXX=clang++"
```

Comment:
Change this to `export CC=${PREFIX}/bin/clang` and similar.

Reply:
Done